pkg/graph/resolvers/plan_basics_test.go

```diff
@@ -87,13 +88,14 @@ func (suite *ResolverSuite) TestUpdatePlanBasics() {
 	suite.NoError(err)
 
 	changes := map[string]interface{}{
-		"modelType":     models.MTVoluntary,
-		"goal":          "Some goal",
-		"cmsCenters":    []string{"CMMI"},
-		"cmmiGroups":    []string{"PATIENT_CARE_MODELS_GROUP", "SEAMLESS_CARE_MODELS_GROUP"},
-		"completeICIP":  "2020-05-13T20:47:50.12Z",
-		"phasedIn":      true,
-		"highLevelNote": "Some high level note",
+		"modelType":      []models.ModelType{models.MTVoluntary},
+		"modelTypeOther": "Some model type other note",
```

Comment:
These two lines are the newly added ones. The other lines are indented to preserve formatting.